policies/step_image_registries.rego

```diff
@@ -11,6 +11,10 @@ deny[{"msg": msg}] {
 
 	some step_index
 	step = att.predicate.buildConfig.steps[step_index]
+
+	# TODO: This is more likely to extract an organization rather than a repo.
```

Comment:
I think it depends on the registry. It's perfectly valid to have an image reference without an "organization". BTW, sometimes the "organization" bit is called "namespace". Sometimes, namespace/repo is called repo... I don't think there's a consistent way of calling these things - it's very much registry dependent.

Reply:
Ack. Maybe I'll revise the comment.

Reply:
Actually I'll take it out since it doesn't belong in this PR..